src/retrieval.rs

```diff
@@ -134,3 +134,11 @@ pub enum FetchOptions {
     All,
     Specific(Vec<EntryHashB64>),
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct FetchEntriesTime {
+    pub year: usize,
```

Comment:
```suggestion
    pub year: usize,
```

how did you come up with the choosing of `usize` for the type here? I'm uncertain of the pros/cons from a fixed size such as i32, i64, etc